addons/hot/functions/fnc_seekerType_SACLOS.sqf

```diff
@@ -0,0 +1,46 @@
+#include "script_component.hpp"
+/*
+ * Author: Brandon (TCVM)
+ * SACLOS seeker
+ *
+ * Arguments:
+ * 1: Guidance Arg Array <ARRAY>
+ * 2: Seeker State <ARRAY>
+ *
+ * Return Value:
+ * Position of wanted missile pos relative to the camera direction <ARRAY>
+ *
+ * Example:
+ * [] call ace_hot_fnc_seekerType_SACLOS
+ *
+ * Public: No
+ */
+params ["", "_args"];
+_args params ["_firedEH", "", "", "_seekerParams", "_stateParams"];
+_firedEH params ["_shooter","_weapon","","","","","_projectile"];
+_seekerParams params ["_seekerAngle"];
+_stateParams params ["", "_seekerStateParams"];
+_seekerStateParams params ["_turretPath", "_memoryPointGunnerOptics"];
+
+private _shooterPos = AGLToASL (_shooter modelToWorld(_shooter selectionPosition _memoryPointGunnerOptics));
+private _projPos = getPosASL _projectile;
+
+private _lookDirection = if !(_shooter isKindOf "CAManBase") then {
+    _shooter weaponDirection ((_shooter weaponsTurret _turretPath) select 0);
```

Comment:
Indfor Wildcat `"I_Heli_light_03_dynamicLoadout_F"`, missile does not track (shooting as gunner)
```
(vehicle player) weaponsTurret [0] = ["Laserdesignator_mounted","ace_hot_3_launcher"]
(vehicle player) weaponDirection "Laserdesignator_mounted" = [0,0,0]
(vehicle player) weaponDirection "ace_hot_3_launcher"  = [0,0,0]
```
```suggestion
    ([_shooter, _turretPath] call ace_common_fnc_getTurretDirection) select 1
```
This seems to work fine, and also has the benifit of working if the vehicle has a weird mixed weapons setup
(gunner and driver both have same weapon)